src/Link0/ProfilerBundle/EventListener/ProfilingEventListener.php

```diff
@@ -8,88 +8,65 @@
 namespace Link0\ProfilerBundle\EventListener;
 
 use Link0\Profiler\PersistenceHandler;
-use Link0\Profiler\Profiler;
-use Symfony\Component\Console\Event\ConsoleTerminateEvent;
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
-use Symfony\Component\Console\Event\ConsoleCommandEvent;
-use Symfony\Component\HttpKernel\TerminableInterface;
+use Link0\ProfilerBundle\Service\ProfilingService;
 
 /**
  * Class ProfilingEventListener
  *
  * Hooks into the symfony event system, and starts and stops the profiler on certain events
  *
- * Start:
- *  - console.command
- *  - kernel.request
- * Stop:
- *  - console.terminate
- *  - kernel.terminate
+ * You can configure this event listener in your services.yml in the following format
+ *
+ * services:
+ *   kernel.listener.link0profilerlistener:
+ *     class: Link0\ProfilerBundle\EventListener\ProfilingEventListener
+ *     param: [@profiling_service]
```

Comment:
is this documentation needed in here? Since it just duplicates code that's available in your bundle and thus decreases maintainability.